src/Microsoft.PowerShell.ConsoleHost/host/msh/ProgressNode.cs

```diff
@@ -432,7 +435,7 @@ internal static bool IsMinimalProgressRenderingEnabled()
                     barLength = barWidth - 1;
                 }
```

Comment:
When reviewing the code, I found this if-block check seems to never be `true`. `PercentComplete >= 0 && PercentComplete < 100`, so `PercentComplete * barWidth / 100` will always be less than `barWidth`. Maybe consider remove this if-block?

Reply:
Looking at the code, `PercentComplete` is allowed to be negative (I think when `PercentComplete` isn't specified, but it should ever be > 100, so removed that check.